src/client.rs

```diff
@@ -29,25 +32,35 @@ enum SubCommand {
     // Version(version::Version),
 }
 
-pub const XDGRUNTIME: &str = "/run/user/1000/nv-leases";
+#[cfg(unix)]
 #[tokio::main]
-#[allow(unused)]
 // This client assumes you use the default lease directory
 async fn main() -> Result<(), Box<dyn std::error::Error>> {
     // This should be moved to somewhere central.  We also need to add override logic.
     let default_uri = String::from("http://[::1]:10000");
```

Comment:
why keep this?

Reply:
The client is not a part of the `netavark_proxy.rs` lib. Therefore it is assumed this file is binary and needs a main function.

Reply:
what does that have to do with with default_uri?